presto-native-execution/CMakeLists.txt

```diff
@@ -106,71 +106,18 @@ set(VELOX_BUILD_TEST_UTILS
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
-set(Boost_USE_MULTITHREADED TRUE)
-find_package(
-  Boost
-  1.66.0
-  REQUIRED
-  program_options
-  context
-  filesystem
-  regex
-  thread
-  system
-  date_time
-  atomic)
-include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
-
-find_package(gflags COMPONENTS shared)
-
-find_library(GLOG glog)
-
-find_library(FMT fmt)
+set(CMAKE_FIND_PACKAGE_TARGETS_GLOBAL ON)
```

Comment:
@assignUser Can you provide feedback on this change?
The problem I am trying to solve is to avoid finding a package twice (once here and once in velox) and also avoid having to modify at both places.
In order to see the `find_package` output from Velox, I added `CMAKE_FIND_PACKAGE_TARGETS_GLOBAL` to make them global.

Reply:
Is there a better solution?

Reply:
This will also require the CMake version to be at least 3.24